ambari-server/src/main/java/org/apache/ambari/server/controller/internal/ExportBlueprintRequest.java

```diff
@@ -154,14 +154,16 @@ private void createHostGroupInfo(Collection<ExportedHostGroup> exportedHostGroup
 
 
   private Stack parseStack(Resource clusterResource) throws InvalidTopologyTemplateException {
-    String[] stackTokens = String.valueOf(clusterResource.getPropertyValue(
-        ClusterResourceProvider.CLUSTER_VERSION_PROPERTY_ID)).split("-");
+    Splitter splitter = Splitter.on('-').limit(2);  // We are only interested to split on the first '-', e.g.
```

Comment:
This logic is already implemented in `StackId`, which I think we should reuse.
(I'm already doing that in my work-in-progress change.)

Reply:
fixed